docs/user/building-images.md

```diff
@@ -0,0 +1,268 @@
+
+```console
+mkdir -p output
+
+sudo podman run --rm -it --privileged --pull=newer \
+    --security-opt label=type:unconfined_t \
+    -v $(pwd)/output:/output \
+    -v /var/lib/containers/storage:/var/lib/containers/storage \
+    quay.io/centos-bootc/bootc-image-builder:latest \
+    --type raw \
+    ${OCI_IMAGE_REPO}:${OCI_IMAGE_TAG}
+```
+
+Once `bootc-image-builder` completes, you can find the disk image under `$(pwd)/output/image/disk.raw`.
+
+Refer to `bootc-image-builder`'s [list of image types](https://github.com/osbuild/bootc-image-builder?tab=readme-ov-file#-image-types) for other supported types.
+
+### Signing and Publishing the OS Disk Image (bootc)
+
+Optionally, you can sign and publish your disk image to your OCI registry, too. This helps unify hosting and distribution. Using manifest lists, you can even keep matching bootc and disk images together:
+
+```console
+sudo podman manifest create ${OCI_IMAGE_REPO}-unified:${OCI_IMAGE_TAG}
+sudo podman manifest add \
+    ${OCI_IMAGE_REPO}-unified:${OCI_IMAGE_TAG} docker://${OCI_IMAGE_REPO}:${OCI_IMAGE_TAG}
+sudo podman manifest add \
+    --artifact --artifact-type application/x-raw-disk-image \
+    --os=linux --arch=x86_64 --annotation disktype=raw \
+    ${OCI_IMAGE_REPO}-unified:${OCI_IMAGE_TAG} $(pwd)/output/image/disk.raw
+sudo podman manifest push --all \
+    ${OCI_IMAGE_REPO}-unified:${OCI_IMAGE_TAG} docker://${OCI_IMAGE_REPO}-unified:${OCI_IMAGE_TAG}
+```
+
+### Further References
+
+For further information and practical examples, refer to:
+
+* Example images in the [Flight Control demos repository](https://github.com/flightctl/flightctl-demos).
+* Flight Control demos repository's [automated build pipeline](https://github.com/flightctl/flightctl-demos/blob/main/.github/workflows/build-bootc.yaml).
+* The Fedora/CentOS bootc project's [community-provided examples](https://gitlab.com/fedora/bootc/examples).
+
+## Considerations for Specific Target Platforms
+
+### Red Hat OpenShift Container Native Virtualization (CNV)
+
+### Red Hat Satellite
+
+### VMware vSphere
+
+When building OS images and disk images for VMware vSphere, follow the [generic process](#building-and-publishing-os-images-and-disk-images) with the following changes:
+
+1. Use late binding of the enrollment endpoint and enrollment certificates, injecting the enrollment certificate or even the whole agent configuration through `cloud-init` when provisioning the virtual device.
+2. Optionally, add the `open-vm-tools` guest tools to the image.
+3. Build a disk image of type "vmdk" instead of "raw".
+
+Create a file named `Containerfile` with the following content to build an OS image based on CentOS Stream 9 that includes the Flight Control agent and VMware guest tools, but no agent configuration:
+
+```console
+FROM quay.io/centos-bootc/centos-bootc:stream9
+
+RUN dnf -y copr enable @redhat-et/flightctl-dev centos-stream-9-x86_64 && \
```

Comment:
In the previous section "### Building the OS Image (bootc)" the copr repo mentioned was `@redhat-et/flightctl`. The command is also different in that one specifies the centos stream including the architecture, and the other doesn't.


Should these be the same command, and generally they should target the non-dev repo?

Reply:
Good catch, thank you! Yes, it should be the non-dev repo. It should also be without the chroot (centos-stream-9-x86_64), but that's something we have to fix still. Will keep it in until the fix lands.